modules/sms_email_reminder/cron_email_notification.php

```diff
@@ -7,11 +7,14 @@
 //
 // Created by:
 // Updated by:	Larry Lart on 10/03/2008
+// Updated by:  Daniel Pflieger on 7/16/2017 daniel@growlingflea.com growlingflea@gmail.com daniel@mi-squared.com
 ////////////////////////////////////////////////////////////////////
 
 // larry :: hack add for command line version
 $_SERVER['REQUEST_URI']=$_SERVER['PHP_SELF'];
 $_SERVER['SERVER_NAME']='localhost';
+$_SERVER['HTTP_HOST']='default';
+$ignoreAuth = true;
```

Comment:
since this script can run from browser without logging in, recommend using the exit() trick on this script (like you did in your prior script, as I recall).